### DISCUSSION TITLE
Issues with Neovim Packer on Redhat 7

### DISCUSSION THREAD

**Discussion by coraxwolf** [wbthomason/packer.nvim#640] [General]
I have a working init.lua with packer.nvim working under both ubuntu and debian, but when I transfered the config files to my work system running Redhat packer won't install any plugins.  I get an error in the logs about no plugin selected. PackerSync completes in less than a second with failed to install all plugins.  No error reasons for this.

I have Neovim 0.6 installed.  I've cleaned out off of the config files and just tried to load packer by itself and only plugin is packer.nvim and the papercolor theme.  This theme works when I used vim-plug to manage plugins, but using packer.nvim it can't install it.

Has anyone had issues using nvim 0.6 with init.lua under redhat.  Would like to have all my workstations keep the same setup.

**Comment by wbthomason** [comment]
Are your `git` versions different across the different systems? This is the most common cause of unexpected errors. Also, there's no information when you press "Enter" on a failed installation line in the final status window?

**Reply by coraxwolf** [reply]
Only thing I can find for an error message is "No Plugin Selected" when I hit enter after doing a PackerUpdate.

The git versions are different.  Work machine is using 1.8 and other systems are on 2.11.  Don't think I can update git on the work machine, but I'll check.  This is the latest/updated version of git for enterprise redhat.  I may have to compile git from source for a personal command like I did with neovim.

**Reply by coraxwolf** [reply]
Updating git fixed the issue.

**Reply by wbthomason** [reply]
Great!
